jobsapp/models.py

```diff
@@ -75,10 +88,12 @@ class Job(models.Model):
     filled = models.BooleanField(
         default=False, verbose_name=_("Filled"), help_text=_("Job position is filled.")
     )
-    salary = InclusiveIntegerRangeField(
-        null=True,
+    salary = models.PositiveIntegerField(
```

Comment:
@francescarpi the previous type of this field, `InclusiveIntegerRangeField`, solved the issue #68. Now we are back again to the same discussion. However, if we end up using a `PositiveIntegerField` we should change the `help_text` because with this new field type, we can't set a minimum and maximum value to salary.